chart/application/templates/deployment.yaml

```diff
@@ -87,14 +87,40 @@ spec:
         - name: PSQL_DB_NAME
           value: {{ trimSuffix "-postgresql" . }}
         {{- end }}
+        {{- if (hasSuffix "-postgresql-standalone" .) }}
+        - name: PSQL_HOSTNAME
+          value: "{{ trimSuffix "-standalone" . }}-hl"
+        - name: PSQL_DATABASE_URL
+          value: postgresql://$(PSQL_USERNAME):$(PSQL_PASSWORD)@$(PSQL_HOSTNAME):$(PSQL_PORT)/$(PSQL_DB_NAME)
+        {{- end }}
+        {{- if (hasSuffix "-postgresql-replication" .) }}
+        - name: PSQL_MASTER_HOSTNAME
+          value: "{{ trimSuffix "-replication" . }}-primary-hl"
+        - name: PSQL_REPLICA_HOSTNAME
+          value: "{{ trimSuffix "-replication" . }}-read-hl"
+        - name: PSQL_DATABASE_URL
+          value: postgresql://$(PSQL_USERNAME):$(PSQL_PASSWORD)@$(PSQL_MASTER_HOSTNAME):$(PSQL_PORT)/$(PSQL_DB_NAME)
+        {{- end }}
+        {{- if (hasSuffix "-mongodb-standalone" .) }}
+        - name: MONGODB_HOSTNAME
+          value: "{{ trimSuffix "-standalone" . }}"
+        - name: MONGODB_DATABASE_URL
+          value: "mongodb://admin:$(MONGODB_ROOT_PASSWORD)@$(MONGODB_HOSTNAME):27017/default"
+        {{- end }}
+        {{- if (hasSuffix "-mongodb-replication" .) }}
+        - name: MONGODB_MASTER_HOSTNAME
+          value: "{{ trimSuffix "-replication" . }}-headless"
+        - name: MONGODB_ARBITER_HOSTNAME
+          value: "{{ trimSuffix "-replication" . }}-arbiter-headless"
+        - name: MONGODB_DATABASE_URL
+          value: "mongodb://admin:$(MONGODB_ROOT_PASSWORD)@$(MONGODB_HOSTNAME):27017/default"
```

Comment:
these might need changes in the docs
https://docs.sidepro.cloud/platform-services/mongodb/

Reply:
Yes, that's intended, I forgot to mention it in the issue, but to be consistent with everything else I figured it made sense to be renamed.